parent/pom.xml

```diff
@@ -823,18 +823,93 @@
+                                    <!-- This is temporarily like this in order to test how it would work -->
+                                    <mode>FILE</mode>
+                                </signing>
+                                <checksum>
+                                    <files>false</files>
+                                </checksum>
+                                <deploy>
+                                    <maven>
+                                        <nexus2>
+                                            <maven-central>
+                                                <active>ALWAYS</active>
+                                                <url>https://oss.sonatype.org/service/local</url>
+                                                <snapshotUrl>https://oss.sonatype.org/content/repositories/snapshots/</snapshotUrl>
+                                                <closeRepository>true</closeRepository>
+                                                <releaseRepository>true</releaseRepository>
+                                                <stagingRepositories>${maven.multiModuleProjectDirectory}/target/staging-deploy</stagingRepositories>
+                                                <artifactOverrides>
+                                                    <artifactOverride>
+                                                        <groupId>org.mapstruct</groupId>
+                                                        <artifactId>mapstruct-jdk8</artifactId>
+                                                        <sourceJar>false</sourceJar>
+                                                        <javadocJar>false</javadocJar>
+                                                    </artifactOverride>
+                                                </artifactOverrides>
+                                            </maven-central>
+                                        </nexus2>
+                                    </maven>
+                                </deploy>
+                                <release>
+                                    <github>
+                                        <tagName>{{projectVersion}}</tagName>
+                                        <releaseName>{{projectVersion}}</releaseName>
+                                        <overwrite>false</overwrite>
+                                        <update>
+                                            <enabled>true</enabled>
+                                        </update>
+                                        <changelog>
+                                            <enabled>true</enabled>
+                                            <external>${maven.multiModuleProjectDirectory}/NEXT_RELEASE_CHANGELOG.md</external>
+                                        </changelog>
+                                    </github>
+                                </release>
+                                <files>
```

Comment:
Alternatively, if these artifacts match a BINARY or JAVA_BINARY distribution then consider using the `<distributions>` section https://jreleaser.org/guide/latest/concepts/distributions/index.html

Reply:
Thanks for the pointer. I was reading up on the binaries, but I couldn't really understand if they are a right fit for this or not. This is non an executable binary, but a zip containing the sources, releases JARs, the Javadoc and the documentation

Reply:
Ah gotcha, then using the `files` section is the way to go.